thin-replica-server/proto/state_snapshot_concord.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 VMware, all rights reserved
+//
+// Concord Client's StateSnapshot Service
+
+syntax = "proto3";
+package com.vmware.concord.state_snapshot_concord;
```

Comment:
`com.vmware.concord.state_snapshot_concord` vs `vmware.concord.client.statesnapshot`

Let's be consistent, I suggest `vmware.concord.statesnapshot`.

FYI: https://cloud.google.com/apis/design/naming_convention#package_names

Reply:
We will have two proto files with the same name then, I wanted to avoid that, i.e., why I went the inconsistent route. Will update.

Reply:
I was getting naming conflicts when using same name for both files. I have renamed the TR `state_snapshot.proto` to `replica_state_snapshot.proto`.